server/build.gradle

```diff
@@ -49,29 +57,24 @@ dependencies {
   implementation 'org.slf4j:slf4j-simple:1.7.30'
 
   // JUnit Jupiter API for testing.
-  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
 
   // JUnit Jupiter Engine for testing.
-  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.6.0'
+  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
 
-  testImplementation 'com.mockrunner:mockrunner-servlet:2.0.1'
+  testImplementation 'com.mockrunner:mockrunner-servlet:2.0.6'
 }
 
 application {
   // Define the main class for the application
-  mainClassName = 'umm3601.Server'
+  mainClass.set("umm3601.Server")
```

Comment:
This changes a now deprecated approach to setting the main class name to the "new improved" approach.